tigerpath/majors_and_certificates/majors/SOC_2018.json

```diff
@@ -36,37 +36,62 @@
+      "explanation": "Sociology majors must take nine courses within the department. Up to two courses can be courses with sociological content that come from other departments. (The Director of Undergraduate Studies has to approve those courses after seeing a syllabus.) Within the nine sociology courses, the following four must be taken: SOC 101: Introduction to Sociology (taught in the fall and the spring and can be taken anytime, even after taking mid- and upper-level sociology courses), SOC 300: Claims and Evidence (taken in the Fall of the junior year to prepare students for the Junior Paper), SOC 301: Statistical Methods in Sociology, SOC 302: Sociological Theory.",
       "req_list": [
         {
           "name": "Required",
           "max_counted": 1,
           "min_needed": "ALL",
-          "explanation": "",
+          "explanation": "Within the nine sociology courses, the following four must be taken: SOC 101: Introduction to Sociology (taught in the fall and the spring and can be taken anytime, even after taking mid- and upper-level sociology courses), SOC 300: Claims and Evidence (taken in the Fall of the junior year to prepare students for the Junior Paper), SOC 301: Statistical Methods in Sociology, SOC 302: Sociological Theory.",
+          "completed_by_semester": 6,
           "course_list": [
             "SOC 101",
             "SOC 300",
-            "POL 345/SOC 305",
+            "SOC 301",
             "SOC 302"
           ]
         },
         {
           "name": "Electives",
           "max_counted": 1,
           "min_needed": 5,
-          "explanation": "",
-          "course_list": [
-            "SOC *"
+          "explanation": "In addition to SOC 101, SOC 300, SOC 301, and SOC 302, students are required to take at least one course at the 300-level or higher in order to graduate.",
+          "req_list": [
+            {
+              "name": null,
+              "max_counted": 4,
+              "min_needed": null,
+              "explanation": null,
+              "course_list": [
+                "SOC 2*"
+              ]
+            },
+            {
+              "name": null,
+              "max_counted": null,
+              "min_needed": null,
+              "explanation": null,
+              "course_list": [
+                "SOC 3*",
+                "SOC 4*",
+                "SOC 5*"
+              ],
+              "excluded_course_list": [
+                "SOC 300",
+                "SOC 301",
+                "SOC 302",
+                "POL 345/SOC 305"
+              ]
+            }
           ]
         }
       ]
     },
     {
       "name": "Independent Work",

```

Comment:
A couple things about this:
- Isn't `POL 345/SOC 305` now being replaced by `SOC 301` ?
- This would then be redundant with the "Required" req, so we could just revert it back to the usual `no_req`, right?

This was probably a mistake when originally encoding this major, but I don't think the department is saying that these two courses constitute a new separate requirement, but rather that students use what they learn from those two courses (which are in the list of required courses above) as tools for their independent work.
So no need to require them again here explicitly.